kuka_sunrise_fri_driver/config/cartesian_impedance_controller_config.yaml

```diff
@@ -0,0 +1,9 @@
+cartesian_impedance_controller:
+  ros__parameters:
+    joints:
+    - CartDOF.X
```

Comment:
joints should be named snake_case (even dummy joints), `cart_dof` would be better I think (or maybe `dummy_cart_joint`)

Reply:
I chose that notation because that is what they use for the stiffness and damping vales in the cartesian impedance mode.  But it's true, that "dummy_cart_joint" sounds more generalized.

Reply:
That is perfect on the java side, but here we should follow the ros2 conventions